certified-connectors/Azure Communication Services MMS/apiDefinition.swagger.json

```diff
@@ -0,0 +1,226 @@
+                            "properties": {
+                                "value": {
+                                    "type": "array",
+                                    "items": {
+                                        "description": "Response for a single recipient.",
+                                        "type": "object",
+                                        "properties": {
+                                            "to": {
+                                                "description": "The recipient's phone number.",
+                                                "type": "string"
+                                            },
+                                            "messageId": {
+                                              "description": "The identifier of the outgoing message. Only present if message accepted.",
+                                              "type": "string"
+                                            },
+                                            "httpStatusCode": {
+                                              "format": "int32",
+                                              "description": "HTTP Status code.",
+                                              "type": "integer"
+                                            },
+                                            "errorMessage": {
+                                              "description": "Optional error message in case of 4xx/5xx/repeatable errors.",
+                                              "type": "string"
+                                            },
+                                            "repeatabilityResult": {
+                                              "description": "The result of a repeatable request with one of the case-insensitive values accepted or rejected.",
+                                              "type": "string",
+                                              "enum": [
+                                                "accepted",
+                                                "rejected"
+                                              ]
+                                            },
+                                            "successful": {
+                                              "description": "Indicates if the message is accepted successfully or not.",
+                                              "type": "boolean"
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    },
+    "definitions": {},
+    "parameters": {},
+    "responses": {},
+    "securityDefinitions": {
+        "api_key": {
+            "type": "apiKey",
+            "in": "header",
+            "name": "Connection-String"
+        }
+    },
+    "security": {},
```

Comment:
Can you please refer the swagger file template and correct the swagger validation error?

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiDefinition.swagger.json

Reply:
The swagger is updated.